examples/notebooks/hyperdrive_fp.ipynb

```diff
@@ -0,0 +1,546 @@
+{
```

Comment:
are you intending for us to maintain the `ipynb` and `py` file? Or should the `ipynb` be deleted?